AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -433,6 +433,8 @@ boston.com#@#.postad
 !#safari_cb_affinity
 !
 @@||noen.at/ads.html
+moregameslike.com#%#//scriptlet("abort-on-property-read", "adbh")
+moregameslike.com#@#.ad-200x200
```

Comment:
this rule is enough, please move it to `! Common safari_cb_affinity(general)` section